### ISSUE TITLE
Change Background/Color of Transition Arrows?

### ISSUE THREAD

**Issue by TonganJedi** [lukasz-galka/ngx-gallery#48]
We have a lot of white images (biological specimen photos), so having white arrows makes them hard to find when trying to change images with a mouse. Is there a way to change the color of the arrows, or even the color of the arrow backgrounds so they can be seen against white images?

I see options for enabling/disabling and changing their shapes, but no color references. Please advise.

**Comment by lukasz-galka** [comment]
You can override default styles using `/deep/` operator. 

```
ngx-gallery /deep/ .ngx-gallery-icon {
  color: red;
  background: blue;
}
```

**Comment by german-e-mas** [comment]
I've read that these operators are going to be deprecated in the future. You can read about it in the [Angular Component Style Guide](https://angular.io/guide/component-styles#deprecated-deep--and-ng-deep). Something to keep in mind.

**Comment by TonganJedi** [comment]
I had hoped there would be an alternative way to get to these, but until they come up with a replacement for this, it'll have to do.

**Comment by german-e-mas** [comment]
I used a hacky workaround to solve this (mainly because I tend to shy away from `/deep/`). 

As my project required using _Material Font_ instead of _Font Awesome_, I made a custom font with just the characters required for the gallery (there are free websites that can do this).

Inside that font's CSS, I included a `circle-icon` class. Then, in the gallery, I passed the arrows as: `arrowNextIcon: 'gallery-arrow-right circle-icon'`, where "gallery-arrow-right" is my custom font.

Here's the result (with a white background)

![arrow](https://user-images.githubusercontent.com/14120807/31397448-747a7732-adbc-11e7-8b38-bc95505c18a8.png)
